marty/js/actions/productSourceActionCreators.js

```diff
@@ -0,0 +1,13 @@
+var Marty = require('marty');
+var ProductConstants = require('../constants/productConstants');
+
+var ProductSourceActionCreators = Marty.createActionCreators({
+  types: {
```

Comment:
@jhollingworth could you please use 4 indent in example? as it's common for this repo - https://github.com/voronianski/flux-comparison/blob/master/.jshintrc#L28